<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_floor3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_floor2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_floor1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_floor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 층: 1층</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
